src/amo/actions/reviews.js

```diff
@@ -42,9 +42,15 @@ export const DELETE_ADDON_REVIEW: 'DELETE_ADDON_REVIEW' = 'DELETE_ADDON_REVIEW';
 export const UNLOAD_ADDON_REVIEWS: 'UNLOAD_ADDON_REVIEWS' =
   'UNLOAD_ADDON_REVIEWS';
 
+export type ReviewAddonType = {|
+  iconUrl?: string,
+  id: number,
+  name?: string,
```

Comment:
Why are some props like `name` and `iconUrl` optional?

Reply:
Good question. I'm sure there must have been a reason I did it like that, but I cannot find one now. I'll change it.